docs/tools/index.md

```diff
@@ -245,7 +255,26 @@ Check out the following pre-built tools that you can use with ADK agents:
 
 </div>
 
-## Build your tools
+## Use pre-built tools with ADK agents
+
+Follow these general steps to include tools in your ADK agents:
+
+1. **Import:** Import the desired tool from the tools module. This is
+   `agents.tools` in Python, `google.golang.org/adk/tool/geminitool` in Go, or
+   `com.google.adk.tools` in Java.
```

Comment:
In ADK TS, tools are imported from the root package, suggest adding the following to this list:

> `@google/adk` in TypeScript

to be consistent with the other languages.

Reply:
thanks! updated